jest.config.tsd.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     name: 'types',
   },
   modulePathIgnorePatterns,
+  reporters: ['default', 'github-actions'],
```

Comment:
should we have a separate `jest.config.tsd.ci.js`?

Reply:
I think there is no need, because locally `github-actions` are anyways disable:

https://github.com/facebook/jest/blob/8784260267190498022cb7e93f9f0bb3d8623311/packages/jest-core/src/TestScheduler.ts#L354-L355

For instance, `jest.config.ci.js` is replacing all reporters. Here the same reporters will work locally and in CI (because of the lines above).